Makefile

```diff
@@ -166,13 +165,4 @@ gen-manifests: prepare-workdir
 	cd $(WORKDIR)
 	@$(CONTAINER_ENGINE) run --rm -v $(WORKDIR):/workdir -w /workdir \
 		$(LINUX_IMAGE) /bin/bash -c "${SCRIPTS_DIR}/gen_manifests.sh $(WORKFLOW_ID) $(ENABLE_PERSISTENCE)"
-	@echo "Manifests are available in workdir $(WORKDIR)/$(WORKFLOW_ID)/manifests"
-
-# Target: push-manifests
```

Comment:
any reasons for removing this target?
as I said before, "escalation" is built with the Makefile and manifests pushed with this target